### PR TITLE
[14.0][MIG] sale_order_digitized_signature

### PR DESCRIPTION
**Pull request by Freni-OSI**

### CODE REVIEW

File: sale_order_digitized_signature/readme/INSTALL.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+This module depends on :
```
Comment (dalonsod) [inline sale_order_digitized_signature/readme/INSTALL.rst]: This should be removed, since `web_widget_digitized_signature` is no more required.

Comment (dalonsod) [inline sale_order_digitized_signature/readme/INSTALL.rst]: This should be removed, since `web_widget_digitized_signature` is no more required.

**Review by dalonsod — COMMENTED** [review]
Locally tested 👍 
Only see comment for `README` info.

**Review by max3903 — APPROVED** [review]

**Comment by max3903** [comment]
@Urvisha-OSI Can you take care of the comments?

**Pull request merged**